cmd/commands/reset/command.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package reset
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/mysteriumnetwork/node/config"
+	"github.com/mysteriumnetwork/node/core/auth"
+	"github.com/mysteriumnetwork/node/core/node"
+	"github.com/mysteriumnetwork/node/core/storage/boltdb"
+	"github.com/urfave/cli/v2"
+)
+
+// flagResetTequilapiAuth instructs to reset Tequilapi auth.
+var flagResetTequilapiAuth = cli.BoolFlag{
+	Name: "tequilapi",
+	Usage: fmt.Sprintf("Reset Tequilapi auth credentials to values provider in --%s and --%s flags",
```

Comment:
If I understand it correctly this is not Tequilapi credentials, this is just WebUI credentials.

Reply:
I'd argue that there is no such thing as WebUI credentials, webUI is just representational layer that display data which it receives via authenticated requests from tequilapi.

Reply:
I'm still can do a call to Tequilapi without any auth like `curl 127.0.0.1:4050/identities`, right?

Reply:
@soffokl named correct issue, just he calls it WebUI credentials. How it is:
```
127.0.0.1:4050/identities - Tequilapi wo auth (controled with `--tequilapi.address`)
127.0.0.1:4999/tequilapi/identities - Tequilapi with auth (controlled with `--ui.address`)
```

Reply:
I can rename it to `--ui.username`, but would go having single tequilapi server + always with authorisation layer